src/identity-providers/add/AddMapper.tsx

```diff
@@ -35,6 +35,8 @@ import { convertToFormValues } from "../../util";
 import { toIdentityProvider } from "../routes/IdentityProvider";
 import type IdentityProviderMapperRepresentation from "@keycloak/keycloak-admin-client/lib/defs/identityProviderMapperRepresentation";
 import { AddMapperForm } from "./AddMapperForm";
+import { useServerInfo } from "../../context/server-info/ServerInfoProvider";
+import _ from "lodash";
```

Comment:
Instead of including all of the Lodash functionality you can also used named imports for the individual functions. This allows the build system to more aggressively [optimize](https://webpack.js.org/guides/tree-shaking/) our code as it can understand which code can be eliminated.
```suggestion
import { groupBy } from "lodash";
```

Reply:
Great idea. This is cool - thank you for linking the tree shaking doc